src_assets/common/assets/web/config.html

```diff
@@ -1051,6 +1064,7 @@ <h1 class="my-4">Configuration</h1>
     "vt_realtime": "enabled",
     "vt_software": "auto",
     "global_prep_cmd": "[]",
+    "max_bitrate": 0,
```

Comment:
These should be alphabetical, please fix the one above yours while you're at it.